modules/liveIntentIdSystem.js

```diff
@@ -5,14 +5,32 @@
  * @requires module:modules/userId
  */
 import * as utils from '../src/utils.js';
-import { ajax } from '../src/ajax.js';
```

Comment:
the default timeout is 3s when the `ajax` is used in prebid. (`ajax=ajaxBuilder(3000)`). Now we create `ajaxBuilder` ourselves and the default timeout in live-connect is 1s. Is it ok or should I change the default timeout in liveconnect to be the same as in prebid?

Reply:
1s is fine